romana/client/client.go

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) 2016 Pani Networks
+// All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License. You may obtain
+// a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations
+// under the License.
+
+// Package client implements glue code for multiple
+// platforms like openstack and kubernetes.
+package client
+
+import (
+	"errors"
+
+	"github.com/romana/core/romana/kubernetes"
+	"github.com/romana/core/romana/openstack"
+
+	"github.com/spf13/viper"
+)
+
+// GetTenantName returns openstack tenant name corresponding
+// to the UUID being used in romana tenants.
+func GetTenantName(uuid string) (string, error) {
+	if platform := viper.GetString("Platform"); platform == "kubernetes" {
```

Comment:
I find the naming of these packages very odd. It seems to me (can't be sure, since it wasn't explained anywhere) that "viper" is related to configs (from files and env vars, etc.). But having never heard of 'viper' before, I find this package name to be very unhelpful.

In Python you can do something like:

```import foo as bar```

So afterwards you can use 'bar' in your code, which hopefully is more meaningful. There is no similar concept in Go?

Furthermore, fancy names such as "viper" should be reserved for big, top-level projects: OpenStack, Django, Redis, Kubernetes, etc. They are well known and big. They also consist of many modules and so they can have fancy code words as names. But smaller utility projects, which are nothing more than helper libraries, would be much better served with meaningful names, rather than fancy ones.

config
runtime-config

Even 'viper-config' would still work, if you really want to use something like that. But 'viper' is just completely devoid of meaning.

Anyway, enough with the ranting. So, is there a "import X as Y" equivalent in Go?